app/models/SessionsRepository.scala

```diff
@@ -33,23 +33,37 @@ case class SessionInfo(userId: String,
                        cancelled: Boolean,
                        active: Boolean,
                        expirationDate: BSONDateTime,
+                       youtubeLink: Option[String],
+                       slideShareLink: Option[String],
                        reminder: Boolean = false,
                        notification: Boolean = false,
-                       _id: BSONObjectID = BSONObjectID.generate)
+                       _id: BSONObjectID = BSONObjectID.generate
+                      )
 
 case class UpdateSessionInfo(sessionUpdateFormData: UpdateSessionInformation,
                              expirationDate: BSONDateTime)
 
+case class SessionLinks(email: String,
```

Comment:
Would it make more sense to rename this case class to SessionURLs?

Reply:
This case class has been removed